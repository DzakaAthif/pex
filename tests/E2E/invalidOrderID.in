BUY 1 Avocado 50 100;
BUY 0 Avocado 50 100;
BUY 2 Avocado 50 110;
BUY 3 Avocado 50 110;
BUY 4 Avocado 50 110;
BUY 1000000 Avocado 50 100;
BUY -1 Avocado 50 100;
BUY -100 Avocado 50 100;
BUY abc Avocado 50 100;
BUY 1.5 Avocado 50 100;
SELL 1 Wine 50 100;
SELL 3 Wine 50 110;
SELL 4 Wine 50 110;
SELL 5 Wine 50 110;
SELL 1000000 Wine 50 100;
SELL -1 Wine 50 100;
SELL -100 Wine 50 100;
SELL abc Wine 50 100;
SELL 1.5 Wine 50 100;
AMEND -1 50 100;
AMEND -100 50 100;
AMEND 1000000 50 100;
AMEND abc 50 100;
AMEND 1.5 50 100;
CANCEL -1;
CANCEL -100;
CANCEL 1000000;
CANCEL abc;
CANCEL 1.5;