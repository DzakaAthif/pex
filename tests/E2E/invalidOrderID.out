[PEX] Starting
[PEX] Trading 2 products: Wine Avocado
[PEX] Created FIFO /tmp/pe_exchange_0
[PEX] Created FIFO /tmp/pe_trader_0
[PEX] Starting trader 0 (./trader1)
[PEX] Connected to /tmp/pe_exchange_0
[PEX] Connected to /tmp/pe_trader_0
[PEX] [T0] Parsing command: <BUY 1 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY 0 Avocado 50 100>
[PEX]	--ORDERBOOK--
[PEX]	Product: Wine; Buy levels: 0; Sell levels: 0
[PEX]	Product: Avocado; Buy levels: 1; Sell levels: 0
[PEX]		BUY 50 @ $100 (1 order)
[PEX]	--POSITIONS--
[PEX]	Trader 0: Wine 0 ($0), Avocado 0 ($0)
[PEX] [T0] Parsing command: <BUY 2 Avocado 50 110>
[PEX] [T0] Parsing command: <BUY 3 Avocado 50 110>
[PEX] [T0] Parsing command: <BUY 4 Avocado 50 110>
[PEX] [T0] Parsing command: <BUY 1000000 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY -1 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY -100 Avocado 50 100>
[PEX] [T0] Parsing command: <BUY abc Avocado 50 100>
[PEX] [T0] Parsing command: <BUY 1.5 Avocado 50 100>
[PEX] [T0] Parsing command: <SELL 1 Wine 50 100>
[PEX]	--ORDERBOOK--
[PEX]	Product: Wine; Buy levels: 0; Sell levels: 1
[PEX]		SELL 50 @ $100 (1 order)
[PEX]	Product: Avocado; Buy levels: 1; Sell levels: 0
[PEX]		BUY 50 @ $100 (1 order)
[PEX]	--POSITIONS--
[PEX]	Trader 0: Wine 0 ($0), Avocado 0 ($0)
[PEX] [T0] Parsing command: <SELL 3 Wine 50 110>
[PEX] [T0] Parsing command: <SELL 4 Wine 50 110>
[PEX] [T0] Parsing command: <SELL 5 Wine 50 110>
[PEX] [T0] Parsing command: <SELL 1000000 Wine 50 100>
[PEX] [T0] Parsing command: <SELL -1 Wine 50 100>
[PEX] [T0] Parsing command: <SELL -100 Wine 50 100>
[PEX] [T0] Parsing command: <SELL abc Wine 50 100>
[PEX] [T0] Parsing command: <SELL 1.5 Wine 50 100>
[PEX] [T0] Parsing command: <AMEND -1 50 100>
[PEX] [T0] Parsing command: <AMEND -100 50 100>
[PEX] [T0] Parsing command: <AMEND 1000000 50 100>
[PEX] [T0] Parsing command: <AMEND abc 50 100>
[PEX] [T0] Parsing command: <AMEND 1.5 50 100>
[PEX] [T0] Parsing command: <CANCEL -1>
[PEX] [T0] Parsing command: <CANCEL -100>
[PEX] [T0] Parsing command: <CANCEL 1000000>
[PEX] [T0] Parsing command: <CANCEL abc>
[PEX] [T0] Parsing command: <CANCEL 1.5>
[PEX] Trader 0 disconnected
[PEX] Trading completed
[PEX] Exchange fees collected: $0
